core/common/src/test/java/alluxio/conf/AlluxioPropertiesTest.java

```diff
@@ -39,6 +40,12 @@
       "value").build();
   private PropertyKey mKeyWithoutValue = new PropertyKey.Builder("key.without.value").build();
 
+  @After
+  public void after() {
+    PropertyKey.unregister(mKeyWithValue);
```

Comment:
for symmetry we could build these in `@Before`

Reply:
done